src/azure-cli/azure/cli/command_modules/keyvault/_params.py

```diff
@@ -89,6 +89,9 @@ class CLIJsonWebKeyOperation(str, Enum):
         c.argument('location', validator=get_default_location_from_resource_group)
         c.argument('enable_soft_delete', arg_type=get_three_state_flag(), help="Enable 'soft delete' functionality for this key vault and all contained entities. If omitted, it will be set to true by default. Once set to true, it cannot be reverted to false.")
         c.argument('retention_days', help='Soft delete data retention days. It accepts >=7 and <=90.', default='90')
+        c.argument('network_acls', help='Network ACLs. It accepts a JSON filename or a JSON string. '
```

Comment:
besides ip and vnet, any other supported rules? how about flatten it as --ips and --vnets?

Reply:
> besides ip and vnet, any other supported rules? how about flatten it as --ips and --vnets?

@yungezz Only IP and VNet rules now, flatten is also a good idea. How about we support `--network-acls <json>`, `--network-acls-ips <ip list>`, `--network-acls-vnets <vnet list>` at the same time? They can work together and all three parameters are optional.